### ISSUE TITLE
Little + next to child folders not showing

### ISSUE THREAD

**Issue by B00ze64** [ariccio/altWinDirStat#12]
Good day.

Windows 7 x64 v0.9.5, when I drill-down into folders, I see the + next to level 1 folders but not next to anything below that. It's better than 0.9.4, which showed a garbled folder list for me, but still has some issues on my system. I could post a screenshot but really, all you would see is the lack of + on child folders. Also, feel free to open another issue but I think a progress bar at the bottom (like original WDS) would be nice to have :-)

Thank you.
Best Regards,

**Comment by hybridview** [comment]
Same issue here on Server 2012 R2.

**Comment by ariccio** [comment]
Hmm, looks like my GitHub notifications aren't making my Gmail Priority Inbox. I've been doing some work on Clang/LLVM, so I haven't checked this page in a while, sorry!

I've noticed this, but haven't actually looked into it yet (duh) since double clicking or using the keyboard still works. Can you still navigate?

I'll check it out.

**Comment by B00ze64** [comment]
Yup, I can still navigate by double-clicking or using the keyboard, so it's nothing major, but until someone double-clicks they may go "hmmm, now what?" ;-)
